staging/src/github.com/openshift/oc/pkg/cli/admin/release/mirror.go

```diff
@@ -442,6 +451,26 @@ func (o *MirrorOptions) Run() error {
 	} else {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
+	
+	mirrorRef, _ := imagereference.Parse(o.To)
+	mirrorRepo := mirrorRef.AsRepository().String()
+
+	sourceRef, _ := imagereference.Parse(o.From)
+	sourceRepo := sourceRef.AsRepository().String()
+
+	fmt.Fprintf(o.Out, "\nTo use the new mirrored release-image to install, provide the following list of release-image sources to the installer using the install-config.yaml:\n")
+	fmt.Fprintf(o.Out, `
+imageContentSource:
+- sources:
+  - %s
+  - %s
+`, mirrorRepo, sourceRepo)
+
+	// Print unordered list of repos mirrored from for use with installing or upgrading when disconnected
+	for repo := range repoMap {
```

Comment:
wouldn't repoMap include the `o.From` already?

Reply:
repoMap only includes (unordered) repos listed in image-references, so o.From is not included.